receiver/windowsservicereceiver/go.mod

```diff
@@ -0,0 +1,56 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/receiver/windowsservicereceiver
+
+go 1.22.0
+
+require (
+	github.com/google/go-cmp v0.6.0
+	github.com/stretchr/testify v1.9.0
+	go.opentelemetry.io/collector/component v0.114.0
```

Comment:
After taking the comments versions should be updated to match the latest of the collector.